<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="es" xml:lang="es"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="OMICs analysis">

<title>Demo Perfil Taxonómico usando Kraken y BioBackery</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="index_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-30a2e6d3c8088394efc257bdd87cc71a.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<style>html{ scroll-behavior: smooth; }</style>


</head>

<body class="quarto-light">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contenido</h2>
   
  <ul>
  <li><a href="#instalaciones-y-configuraciones" id="toc-instalaciones-y-configuraciones" class="nav-link active" data-scroll-target="#instalaciones-y-configuraciones">Instalaciones y configuraciones</a>
  <ul class="collapse">
  <li><a href="#kraken2" id="toc-kraken2" class="nav-link" data-scroll-target="#kraken2">Kraken2</a></li>
  <li><a href="#bracken" id="toc-bracken" class="nav-link" data-scroll-target="#bracken">Bracken</a></li>
  <li><a href="#humann-y-metaphlan" id="toc-humann-y-metaphlan" class="nav-link" data-scroll-target="#humann-y-metaphlan">HUMAnN y MetaPhlAn</a></li>
  <li><a href="#humann-data" id="toc-humann-data" class="nav-link" data-scroll-target="#humann-data">HUMAnN data</a></li>
  <li><a href="#metaphlan-data" id="toc-metaphlan-data" class="nav-link" data-scroll-target="#metaphlan-data">MetaPhlAn data</a></li>
  <li><a href="#otros-softwares" id="toc-otros-softwares" class="nav-link" data-scroll-target="#otros-softwares">Otros softwares:</a>
  <ul class="collapse">
  <li><a href="#sratools" id="toc-sratools" class="nav-link" data-scroll-target="#sratools">SRATools</a></li>
  <li><a href="#kneaddata" id="toc-kneaddata" class="nav-link" data-scroll-target="#kneaddata">KneadData</a></li>
  <li><a href="#flash" id="toc-flash" class="nav-link" data-scroll-target="#flash">Flash</a></li>
  <li><a href="#trimmomatic" id="toc-trimmomatic" class="nav-link" data-scroll-target="#trimmomatic">Trimmomatic</a></li>
  <li><a href="#cutadapt" id="toc-cutadapt" class="nav-link" data-scroll-target="#cutadapt">CutAdapt</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#conjunto-de-datos" id="toc-conjunto-de-datos" class="nav-link" data-scroll-target="#conjunto-de-datos">Conjunto de datos</a>
  <ul class="collapse">
  <li><a href="#descarga-de-los-datos" id="toc-descarga-de-los-datos" class="nav-link" data-scroll-target="#descarga-de-los-datos">Descarga de los datos</a></li>
  <li><a href="#estructura-proyecto" id="toc-estructura-proyecto" class="nav-link" data-scroll-target="#estructura-proyecto">Estructura proyecto</a></li>
  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a></li>
  <li><a href="#runs" id="toc-runs" class="nav-link" data-scroll-target="#runs">RUNs</a></li>
  <li><a href="#preprocemiento" id="toc-preprocemiento" class="nav-link" data-scroll-target="#preprocemiento">Preprocemiento</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Demo Perfil Taxonómico usando Kraken y BioBackery</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Autor/a</div>
    <div class="quarto-title-meta-contents">
             <p>OMICs analysis </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>

<nav id="TOC-body" role="doc-toc">
    <h2 id="toc-title">Contenido</h2>
   
  <ul>
  <li><a href="#instalaciones-y-configuraciones" id="toc-instalaciones-y-configuraciones">Instalaciones y configuraciones</a>
  <ul>
  <li><a href="#kraken2" id="toc-kraken2">Kraken2</a></li>
  <li><a href="#bracken" id="toc-bracken">Bracken</a></li>
  <li><a href="#humann-y-metaphlan" id="toc-humann-y-metaphlan">HUMAnN y MetaPhlAn</a></li>
  <li><a href="#humann-data" id="toc-humann-data">HUMAnN data</a></li>
  <li><a href="#metaphlan-data" id="toc-metaphlan-data">MetaPhlAn data</a></li>
  <li><a href="#otros-softwares" id="toc-otros-softwares">Otros softwares:</a>
  <ul>
  <li><a href="#sratools" id="toc-sratools">SRATools</a></li>
  <li><a href="#kneaddata" id="toc-kneaddata">KneadData</a></li>
  <li><a href="#flash" id="toc-flash">Flash</a></li>
  <li><a href="#trimmomatic" id="toc-trimmomatic">Trimmomatic</a></li>
  <li><a href="#cutadapt" id="toc-cutadapt">CutAdapt</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#conjunto-de-datos" id="toc-conjunto-de-datos">Conjunto de datos</a>
  <ul>
  <li><a href="#descarga-de-los-datos" id="toc-descarga-de-los-datos">Descarga de los datos</a></li>
  <li><a href="#estructura-proyecto" id="toc-estructura-proyecto">Estructura proyecto</a></li>
  <li><a href="#data" id="toc-data">Data</a></li>
  <li><a href="#runs" id="toc-runs">RUNs</a></li>
  <li><a href="#preprocemiento" id="toc-preprocemiento">Preprocemiento</a></li>
  </ul></li>
  </ul>
</nav>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/logo_OAN_for_white.png" class="img-fluid quarto-figure quarto-figure-center figure-img" width="300"></p>
</figure>
</div>
<section id="instalaciones-y-configuraciones" class="level1">
<h1>Instalaciones y configuraciones</h1>
<section id="kraken2" class="level3">
<h3 class="anchored" data-anchor-id="kraken2">Kraken2</h3>
<p>Se realizaron las siguientes descargas directamente desde la terminal:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sudo</span> apt update</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">sudo</span> apt install build-essential g++ libomp-dev wget rsync perl git</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">sudo</span> apt install zlib1g-dev</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Posteriormente se clonó el repositorio de <code>Kraken</code> para poder instalarlo:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> old </span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">git</span> clone https://github.com/DerrickWood/kraken2.git</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> kraken2</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="fu">mkdir</span> <span class="at">-p</span> /home/ubuntu/bin/kraken2</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="ex">./install_kraken2.sh</span> /home/ubuntu/bin/kraken2</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>La base de datos que se utilizó ya venía previamente indexada, lo cual nos ahorró el paso de generarla desde cero, esta base datos es <code>Estándar</code> y contiene lo siguiente: <code>Refseq archaea, bacteria, viral, plasmid, human, UniVec_Core</code> y es de fecha 07/14/2025.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">mkdir</span> <span class="at">-p</span> /home/ubuntu/data/db_standard</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> /home/ubuntu/data/db_standard </span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="fu">wget</span> https://genome-idx.s3.amazonaws.com/kraken/k2_standard_20250714.tar.gz</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="fu">tar</span> <span class="at">-xzf</span> k2_standard_20250714.tar.gz</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Nota: una vez que se termina de descomprimir hay que eliminar el archivo <code>k2_standard_20250714.tar.gz</code> para ahorrar espacio.</p>
</section>
<section id="bracken" class="level3">
<h3 class="anchored" data-anchor-id="bracken">Bracken</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> /home/ubuntu/bin</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="fu">git</span> clone https://github.com/jenniferlu717/Bracken.git</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> src/ <span class="kw">&amp;&amp;</span> <span class="fu">make</span></span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="humann-y-metaphlan" class="level3">
<h3 class="anchored" data-anchor-id="humann-y-metaphlan">HUMAnN y MetaPhlAn</h3>
<p>La combinación estable y compatible es HUMAnN 3.8 con MetaPhlAn 4.1 (hay que evitar versiones más nuevas como MetaPhlAn 4.2, que rompen la integración)</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="ex">/home/ubuntu/miniconda3/bin/conda</span> config <span class="at">--add</span> channels defaults</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="ex">/home/ubuntu/miniconda3/bin/conda</span> config <span class="at">--add</span> channels bioconda</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="ex">/home/ubuntu/miniconda3/bin/conda</span> config <span class="at">--add</span> channels conda-forge</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="ex">/home/ubuntu/miniconda3/bin/conda</span> config <span class="at">--add</span> channels biobakery</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="ex">/home/ubuntu/miniconda3/bin/conda</span> create <span class="at">-n</span> bakery_env <span class="at">-y</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="ex">/home/ubuntu/miniconda3/bin/conda</span> init bash</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="bu">source</span> ~/.bashrc</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="ex">conda</span> activate bakery_env</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="ex">conda</span> install humann=3.8 metaphlan=4.1 <span class="at">-y</span></span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Preparación de sus respectivas bases de datos:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">mkdir</span> <span class="at">-p</span> /home/ubuntu/data/humann_db    </span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a><span class="fu">mkdir</span> <span class="at">-p</span> /home/ubuntu/data/metaphlan_db </span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="humann-data" class="level3">
<h3 class="anchored" data-anchor-id="humann-data">HUMAnN data</h3>
<p>ChocoPhlAn (completa):</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="ex">humann_databases</span> <span class="at">--download</span> chocophlan full /home/ubuntu/data/humann_db <span class="at">--update-config</span> yes </span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Uniref90:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="ex">humann_databases</span> <span class="at">--download</span> uniref uniref90_diamond /home/ubuntu/data/humann_db <span class="at">--update-config</span> yes</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Utility mapping:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="ex">humann_databases</span> <span class="at">--download</span> utility_mapping full /home/ubuntu/data/humann_db <span class="at">--update-config</span> yes</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="metaphlan-data" class="level3">
<h3 class="anchored" data-anchor-id="metaphlan-data">MetaPhlAn data</h3>
<p>Se usó el índice compatible mpa_vOct22_CHOCOPhlAnSGB_202403 (integración con HUMAnN 3.8):</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="ex">metaphlan</span> <span class="at">--install</span> <span class="at">--bowtie2db</span> <span class="at">--index</span> mpa_vOct22_CHOCOPhlAnSGB_202403</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="otros-softwares" class="level2">
<h2 class="anchored" data-anchor-id="otros-softwares">Otros softwares:</h2>
<section id="sratools" class="level3">
<h3 class="anchored" data-anchor-id="sratools">SRATools</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> /home/ubuntu/bin</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a><span class="fu">wget</span> <span class="at">--output-document</span> sratoolkit.tar.gz https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-ubuntu64.tar.gz</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="fu">tar</span> <span class="at">-vxzf</span> sratoolkit.tar.gz</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a><span class="bu">export</span> <span class="va">PATH</span><span class="op">=</span><span class="va">$PWD</span>/sratoolkit.3.2.1-ubuntu64/bin:<span class="va">$PATH</span></span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="kneaddata" class="level3">
<h3 class="anchored" data-anchor-id="kneaddata">KneadData</h3>
<p>Se utilizó esta herramienta para hacer el proceso de <code>Trimmomatic</code> y quitar el ADN de hospedero (humano), se puede personalizar al genoma de hospedero que convenga. Es recomendable instalarla en el mismo entorno Conda (<code>bakery_env</code>) para mantener la compatibilidad y simplicidad.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="ex">conda</span> install <span class="at">-c</span> bioconda kneaddata <span class="at">-y</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a><span class="fu">mkdir</span> <span class="at">-p</span> /home/ubuntu/data/kneaddata_db</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a><span class="ex">kneaddata_database</span> <span class="at">--download</span> human_genome bowtie2 /home/ubuntu/data/kneaddata_db</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="flash" class="level3">
<h3 class="anchored" data-anchor-id="flash">Flash</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="trimmomatic" class="level3">
<h3 class="anchored" data-anchor-id="trimmomatic">Trimmomatic</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="bu">cd</span> /home/ubuntu/bin</span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a><span class="fu">wget</span> http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.39.zip</span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a><span class="fu">unzip</span> Trimmomatic-0.39.zip</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="cutadapt" class="level3">
<h3 class="anchored" data-anchor-id="cutadapt">CutAdapt</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sudo</span> apt install cutadapt</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
</section>
<section id="conjunto-de-datos" class="level1">
<h1>Conjunto de datos</h1>
<p>Para este <code>Demo</code> utilizamos metagenomas de intestino de humano del estudio “<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10680783/#S8">Gut microbiome composition may be an indicator of preclinical Alzheimer’s disease</a>”. Seleccionamos 3 muestras para 2 condiciones distintas, <code>Sintomas Pre-Clínicos de Alzheimer</code> vs <code>Pacientes Sanos</code>.</p>
<p>Aqui están los metadatos asociados a cada muestra:</p>
<div class="cell">
<div class="cell-output-display">
<div id="ftsnsuxjfl" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#ftsnsuxjfl table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#ftsnsuxjfl thead, #ftsnsuxjfl tbody, #ftsnsuxjfl tfoot, #ftsnsuxjfl tr, #ftsnsuxjfl td, #ftsnsuxjfl th {
  border-style: none;
}

#ftsnsuxjfl p {
  margin: 0;
  padding: 0;
}

#ftsnsuxjfl .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ftsnsuxjfl .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#ftsnsuxjfl .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ftsnsuxjfl .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ftsnsuxjfl .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ftsnsuxjfl .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ftsnsuxjfl .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ftsnsuxjfl .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ftsnsuxjfl .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ftsnsuxjfl .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ftsnsuxjfl .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ftsnsuxjfl .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ftsnsuxjfl .gt_spanner_row {
  border-bottom-style: hidden;
}

#ftsnsuxjfl .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#ftsnsuxjfl .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ftsnsuxjfl .gt_from_md > :first-child {
  margin-top: 0;
}

#ftsnsuxjfl .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ftsnsuxjfl .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ftsnsuxjfl .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#ftsnsuxjfl .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#ftsnsuxjfl .gt_row_group_first td {
  border-top-width: 2px;
}

#ftsnsuxjfl .gt_row_group_first th {
  border-top-width: 2px;
}

#ftsnsuxjfl .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ftsnsuxjfl .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#ftsnsuxjfl .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#ftsnsuxjfl .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ftsnsuxjfl .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ftsnsuxjfl .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ftsnsuxjfl .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#ftsnsuxjfl .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ftsnsuxjfl .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ftsnsuxjfl .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ftsnsuxjfl .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ftsnsuxjfl .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ftsnsuxjfl .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ftsnsuxjfl .gt_left {
  text-align: left;
}

#ftsnsuxjfl .gt_center {
  text-align: center;
}

#ftsnsuxjfl .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ftsnsuxjfl .gt_font_normal {
  font-weight: normal;
}

#ftsnsuxjfl .gt_font_bold {
  font-weight: bold;
}

#ftsnsuxjfl .gt_font_italic {
  font-style: italic;
}

#ftsnsuxjfl .gt_super {
  font-size: 65%;
}

#ftsnsuxjfl .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#ftsnsuxjfl .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#ftsnsuxjfl .gt_indent_1 {
  text-indent: 5px;
}

#ftsnsuxjfl .gt_indent_2 {
  text-indent: 10px;
}

#ftsnsuxjfl .gt_indent_3 {
  text-indent: 15px;
}

#ftsnsuxjfl .gt_indent_4 {
  text-indent: 20px;
}

#ftsnsuxjfl .gt_indent_5 {
  text-indent: 25px;
}

#ftsnsuxjfl .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#ftsnsuxjfl div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_col_headings header">
<th id="Run" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Run</th>
<th id="Assay-Type" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Assay Type</th>
<th id="AvgSpotLen" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">AvgSpotLen</th>
<th id="Bases" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Bases</th>
<th id="BioProject" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">BioProject</th>
<th id="BioSample" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">BioSample</th>
<th id="BioSampleModel" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">BioSampleModel</th>
<th id="Bytes" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Bytes</th>
<th id="Center-Name" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Center Name</th>
<th id="Collection_Date" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Collection_Date</th>
<th id="Consent" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Consent</th>
<th id="DATASTORE-filetype" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">DATASTORE filetype</th>
<th id="DATASTORE-provider" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">DATASTORE provider</th>
<th id="DATASTORE-region" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">DATASTORE region</th>
<th id="Experiment" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Experiment</th>
<th id="geo_loc_name_country" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">geo_loc_name_country</th>
<th id="geo_loc_name_country_continent" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">geo_loc_name_country_continent</th>
<th id="geo_loc_name" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">geo_loc_name</th>
<th id="HOST" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">HOST</th>
<th id="Instrument" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Instrument</th>
<th id="isolation_source" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">isolation_source</th>
<th id="lat_lon" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">lat_lon</th>
<th id="Library-Name" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Library Name</th>
<th id="LibraryLayout" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">LibraryLayout</th>
<th id="LibrarySelection" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">LibrarySelection</th>
<th id="LibrarySource" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">LibrarySource</th>
<th id="Organism" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Organism</th>
<th id="Platform" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Platform</th>
<th id="ReleaseDate" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">ReleaseDate</th>
<th id="create_date" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">create_date</th>
<th id="version" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">version</th>
<th id="Sample-Name" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Sample Name</th>
<th id="SRA-Study" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">SRA Study</th>
<th id="Age" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Age</th>
<th id="Sex" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Sex</th>
<th id="Race" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Race</th>
<th id="Years-Education" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Years Education</th>
<th id="CDR" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CDR</th>
<th id="LUMIPULSE_CSF.ab42.ab40.ratio" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">LUMIPULSE_CSF.ab42.ab40.ratio</th>
<th id="LUMIPULSE_CSF_tTau" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">LUMIPULSE_CSF_tTau</th>
<th id="LUMIPULSE_CSF_pTau" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">LUMIPULSE_CSF_pTau</th>
<th id="Days.SincePETpib" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Days.SincePETpib</th>
<th id="pib_fsuvr_rsf_tot_cortmean" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">pib_fsuvr_rsf_tot_cortmean</th>
<th id="Centiloid.PiB" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Centiloid.PiB</th>
<th id="Days.SincePETav45" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Days.SincePETav45</th>
<th id="av45_fsuvr_rsf_tot_cortmean" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">av45_fsuvr_rsf_tot_cortmean</th>
<th id="Centiloid.AV45" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Centiloid.AV45</th>
<th id="Tracer2Use" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tracer2Use</th>
<th id="combined.Centiloid" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">combined.Centiloid</th>
<th id="amyloid.positive.AF" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">amyloid.positive.AF</th>
<th id="Tauopathy" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tauopathy</th>
<th id="tau.positive.AF" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">tau.positive.AF</th>
<th id="CortSig_Thickness" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CortSig_Thickness</th>
<th id="MR_LV_RV_HIPPOCAMPUS" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MR_LV_RV_HIPPOCAMPUS</th>
<th id="WMH_volume" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">WMH_volume</th>
<th id="APOE" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">APOE</th>
<th id="APOE4.status" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">APOE4.status</th>
<th id="Polygenic-Risk-Score" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Polygenic Risk Score</th>
<th id="INT_PET" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">INT_PET</th>
<th id="INT_CSF" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">INT_CSF</th>
<th id="INT_MRI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">INT_MRI</th>
<th id="INT_CDR" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">INT_CDR</th>
<th id="Interval_Measure_Amyloid" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Interval_Measure_Amyloid</th>
<th id="Interval_Days_Amyloid" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Interval_Days_Amyloid</th>
<th id="WEIGHT.lb" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">WEIGHT.lb</th>
<th id="HEIGHT.in" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">HEIGHT.in</th>
<th id="BMI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">BMI</th>
<th id="Cancer" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Cancer</th>
<th id="Active.Depression" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Active.Depression</th>
<th id="Hypercholesterolemia" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Hypercholesterolemia</th>
<th id="Hypertension" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Hypertension</th>
<th id="Diabetes" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Diabetes</th>
<th id="Liver.Disease" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Liver.Disease</th>
<th id="Alcohol.Abuse" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Alcohol.Abuse</th>
<th id="Tobacco.Use.PastOrPresent" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Tobacco.Use.PastOrPresent</th>
<th id="Autoimmune.Disorder" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Autoimmune.Disorder</th>
<th id="Thyroid.Disease" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Thyroid.Disease</th>
<th id="Cardiovascular.Disease" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Cardiovascular.Disease</th>
<th id="Reads" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Reads</th>
<th id="Richness" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Richness</th>
<th id="Shannon" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Shannon</th>
<th id="Tax.PCoA1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Tax.PCoA1</th>
<th id="Tax.PCoA2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Tax.PCoA2</th>
<th id="FBratio" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">FBratio</th>
<th id="Fnl.Richness" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Fnl.Richness</th>
<th id="Fnl.Shannon" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Fnl.Shannon</th>
<th id="Fnl.PCoA1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Fnl.PCoA1</th>
<th id="Fnl.PCoA2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Fnl.PCoA2</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Run">SRR17648378</td>
<td class="gt_row gt_left" headers="Assay Type">WGS</td>
<td class="gt_row gt_right" headers="AvgSpotLen">239</td>
<td class="gt_row gt_right" headers="Bases">4022515981</td>
<td class="gt_row gt_left" headers="BioProject">PRJNA798058</td>
<td class="gt_row gt_left" headers="BioSample">SAMN25046600</td>
<td class="gt_row gt_left" headers="BioSampleModel">Metagenome or environmental</td>
<td class="gt_row gt_right" headers="Bytes">1390650146</td>
<td class="gt_row gt_left" headers="Center Name">WASHINGTON UNIVERSITY IN ST. LOUIS</td>
<td class="gt_row gt_right" headers="Collection_Date">2020</td>
<td class="gt_row gt_left" headers="Consent">public</td>
<td class="gt_row gt_left" headers="DATASTORE filetype">fastq,run.zq,sra</td>
<td class="gt_row gt_left" headers="DATASTORE provider">gs,ncbi,s3</td>
<td class="gt_row gt_left" headers="DATASTORE region">gs.us-east1,ncbi.public,s3.us-east-1</td>
<td class="gt_row gt_left" headers="Experiment">SRX13816679</td>
<td class="gt_row gt_left" headers="geo_loc_name_country">USA</td>
<td class="gt_row gt_left" headers="geo_loc_name_country_continent">North America</td>
<td class="gt_row gt_left" headers="geo_loc_name">USA: St. Louis</td>
<td class="gt_row gt_left" headers="HOST">Homo sapiens</td>
<td class="gt_row gt_left" headers="Instrument">Illumina NovaSeq 6000</td>
<td class="gt_row gt_left" headers="isolation_source">Stool 8</td>
<td class="gt_row gt_left" headers="lat_lon">38.6270 N 90.1994 W</td>
<td class="gt_row gt_left" headers="Library Name">Stool 8</td>
<td class="gt_row gt_left" headers="LibraryLayout">PAIRED</td>
<td class="gt_row gt_left" headers="LibrarySelection">RANDOM</td>
<td class="gt_row gt_left" headers="LibrarySource">METAGENOMIC</td>
<td class="gt_row gt_left" headers="Organism">human gut metagenome</td>
<td class="gt_row gt_left" headers="Platform">ILLUMINA</td>
<td class="gt_row gt_right" headers="ReleaseDate">2022-05-01</td>
<td class="gt_row gt_right" headers="create_date">2022-01-17 19:23:00</td>
<td class="gt_row gt_right" headers="version">1</td>
<td class="gt_row gt_right" headers="Sample Name">62568</td>
<td class="gt_row gt_left" headers="SRA Study">SRP355475</td>
<td class="gt_row gt_right" headers="Age">85.70137</td>
<td class="gt_row gt_right" headers="Sex">2</td>
<td class="gt_row gt_right" headers="Race">4</td>
<td class="gt_row gt_right" headers="Years Education">18</td>
<td class="gt_row gt_right" headers="CDR">0</td>
<td class="gt_row gt_left" headers="LUMIPULSE_CSF.ab42.ab40.ratio">8.7997142E-2</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_tTau">209</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_pTau">29.2</td>
<td class="gt_row gt_left" headers="Days.SincePETpib">NA</td>
<td class="gt_row gt_left" headers="pib_fsuvr_rsf_tot_cortmean">NA</td>
<td class="gt_row gt_left" headers="Centiloid.PiB">NA</td>
<td class="gt_row gt_left" headers="Days.SincePETav45">2331</td>
<td class="gt_row gt_left" headers="av45_fsuvr_rsf_tot_cortmean">0.93899999999999995</td>
<td class="gt_row gt_left" headers="Centiloid.AV45">7.1303999999999901</td>
<td class="gt_row gt_left" headers="Tracer2Use">AV45</td>
<td class="gt_row gt_right" headers="combined.Centiloid">7.1303999999999901</td>
<td class="gt_row gt_right" headers="amyloid.positive.AF">0</td>
<td class="gt_row gt_left" headers="Tauopathy">NA</td>
<td class="gt_row gt_right" headers="tau.positive.AF">0</td>
<td class="gt_row gt_right" headers="CortSig_Thickness">2.1889635489999999</td>
<td class="gt_row gt_right" headers="MR_LV_RV_HIPPOCAMPUS">6921.4</td>
<td class="gt_row gt_right" headers="WMH_volume">36641.46875</td>
<td class="gt_row gt_right" headers="APOE">24</td>
<td class="gt_row gt_left" headers="APOE4.status">e4+</td>
<td class="gt_row gt_left" headers="Polygenic Risk Score">-1.3089699999999999E-2</td>
<td class="gt_row gt_right" headers="INT_PET">1516</td>
<td class="gt_row gt_right" headers="INT_CSF">192</td>
<td class="gt_row gt_right" headers="INT_MRI">709</td>
<td class="gt_row gt_right" headers="INT_CDR">202</td>
<td class="gt_row gt_left" headers="Interval_Measure_Amyloid">INT_PET</td>
<td class="gt_row gt_right" headers="Interval_Days_Amyloid">1516</td>
<td class="gt_row gt_right" headers="WEIGHT.lb">173.00000</td>
<td class="gt_row gt_right" headers="HEIGHT.in">63.00</td>
<td class="gt_row gt_right" headers="BMI">30.64526</td>
<td class="gt_row gt_right" headers="Cancer">0</td>
<td class="gt_row gt_right" headers="Active.Depression">0</td>
<td class="gt_row gt_right" headers="Hypercholesterolemia">1</td>
<td class="gt_row gt_right" headers="Hypertension">1</td>
<td class="gt_row gt_right" headers="Diabetes">0</td>
<td class="gt_row gt_right" headers="Liver.Disease">0</td>
<td class="gt_row gt_right" headers="Alcohol.Abuse">0</td>
<td class="gt_row gt_right" headers="Tobacco.Use.PastOrPresent">0</td>
<td class="gt_row gt_right" headers="Autoimmune.Disorder">0</td>
<td class="gt_row gt_right" headers="Thyroid.Disease">1</td>
<td class="gt_row gt_right" headers="Cardiovascular.Disease">0</td>
<td class="gt_row gt_right" headers="Reads">16772435</td>
<td class="gt_row gt_right" headers="Richness">101</td>
<td class="gt_row gt_right" headers="Shannon">2.805364</td>
<td class="gt_row gt_right" headers="Tax.PCoA1">0.005836659</td>
<td class="gt_row gt_right" headers="Tax.PCoA2">-0.15794836</td>
<td class="gt_row gt_right" headers="FBratio">0.89536350174081303</td>
<td class="gt_row gt_right" headers="Fnl.Richness">307</td>
<td class="gt_row gt_right" headers="Fnl.Shannon">5.224984</td>
<td class="gt_row gt_right" headers="Fnl.PCoA1">-0.012065332</td>
<td class="gt_row gt_right" headers="Fnl.PCoA2">0.016554060</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Run">SRR17648430</td>
<td class="gt_row gt_left" headers="Assay Type">WGS</td>
<td class="gt_row gt_right" headers="AvgSpotLen">253</td>
<td class="gt_row gt_right" headers="Bases">12010606930</td>
<td class="gt_row gt_left" headers="BioProject">PRJNA798058</td>
<td class="gt_row gt_left" headers="BioSample">SAMN25046615</td>
<td class="gt_row gt_left" headers="BioSampleModel">Metagenome or environmental</td>
<td class="gt_row gt_right" headers="Bytes">4322857319</td>
<td class="gt_row gt_left" headers="Center Name">WASHINGTON UNIVERSITY IN ST. LOUIS</td>
<td class="gt_row gt_right" headers="Collection_Date">2020</td>
<td class="gt_row gt_left" headers="Consent">public</td>
<td class="gt_row gt_left" headers="DATASTORE filetype">fastq,run.zq,sra</td>
<td class="gt_row gt_left" headers="DATASTORE provider">gs,ncbi,s3</td>
<td class="gt_row gt_left" headers="DATASTORE region">gs.us-east1,ncbi.public,s3.us-east-1</td>
<td class="gt_row gt_left" headers="Experiment">SRX13816627</td>
<td class="gt_row gt_left" headers="geo_loc_name_country">USA</td>
<td class="gt_row gt_left" headers="geo_loc_name_country_continent">North America</td>
<td class="gt_row gt_left" headers="geo_loc_name">USA: St. Louis</td>
<td class="gt_row gt_left" headers="HOST">Homo sapiens</td>
<td class="gt_row gt_left" headers="Instrument">Illumina NovaSeq 6000</td>
<td class="gt_row gt_left" headers="isolation_source">Stool 23</td>
<td class="gt_row gt_left" headers="lat_lon">38.6270 N 90.1994 W</td>
<td class="gt_row gt_left" headers="Library Name">Stool 23</td>
<td class="gt_row gt_left" headers="LibraryLayout">PAIRED</td>
<td class="gt_row gt_left" headers="LibrarySelection">RANDOM</td>
<td class="gt_row gt_left" headers="LibrarySource">METAGENOMIC</td>
<td class="gt_row gt_left" headers="Organism">human gut metagenome</td>
<td class="gt_row gt_left" headers="Platform">ILLUMINA</td>
<td class="gt_row gt_right" headers="ReleaseDate">2022-05-01</td>
<td class="gt_row gt_right" headers="create_date">2022-01-17 20:37:00</td>
<td class="gt_row gt_right" headers="version">1</td>
<td class="gt_row gt_right" headers="Sample Name">63972</td>
<td class="gt_row gt_left" headers="SRA Study">SRP355475</td>
<td class="gt_row gt_right" headers="Age">84.15616</td>
<td class="gt_row gt_right" headers="Sex">2</td>
<td class="gt_row gt_right" headers="Race">5</td>
<td class="gt_row gt_right" headers="Years Education">12</td>
<td class="gt_row gt_right" headers="CDR">0</td>
<td class="gt_row gt_left" headers="LUMIPULSE_CSF.ab42.ab40.ratio">7.0503435000000003E-2</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_tTau">207</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_pTau">29.9</td>
<td class="gt_row gt_left" headers="Days.SincePETpib">2395</td>
<td class="gt_row gt_left" headers="pib_fsuvr_rsf_tot_cortmean">1.1160000000000001</td>
<td class="gt_row gt_left" headers="Centiloid.PiB">2.72000000000001</td>
<td class="gt_row gt_left" headers="Days.SincePETav45">3394</td>
<td class="gt_row gt_left" headers="av45_fsuvr_rsf_tot_cortmean">0.81799999999999995</td>
<td class="gt_row gt_left" headers="Centiloid.AV45">0.64479999999999604</td>
<td class="gt_row gt_left" headers="Tracer2Use">PiB</td>
<td class="gt_row gt_right" headers="combined.Centiloid">2.72000000000001</td>
<td class="gt_row gt_right" headers="amyloid.positive.AF">0</td>
<td class="gt_row gt_left" headers="Tauopathy">NA</td>
<td class="gt_row gt_right" headers="tau.positive.AF">0</td>
<td class="gt_row gt_right" headers="CortSig_Thickness">2.5124870590000001</td>
<td class="gt_row gt_right" headers="MR_LV_RV_HIPPOCAMPUS">7789.5</td>
<td class="gt_row gt_right" headers="WMH_volume">17119.011719999999</td>
<td class="gt_row gt_right" headers="APOE">33</td>
<td class="gt_row gt_left" headers="APOE4.status">e4-</td>
<td class="gt_row gt_left" headers="Polygenic Risk Score">6.5859500000000001E-3</td>
<td class="gt_row gt_right" headers="INT_PET">1550</td>
<td class="gt_row gt_right" headers="INT_CSF">1600</td>
<td class="gt_row gt_right" headers="INT_MRI">1550</td>
<td class="gt_row gt_right" headers="INT_CDR">136</td>
<td class="gt_row gt_left" headers="Interval_Measure_Amyloid">INT_PET</td>
<td class="gt_row gt_right" headers="Interval_Days_Amyloid">1550</td>
<td class="gt_row gt_right" headers="WEIGHT.lb">190.19995</td>
<td class="gt_row gt_right" headers="HEIGHT.in">61.00</td>
<td class="gt_row gt_right" headers="BMI">35.93760</td>
<td class="gt_row gt_right" headers="Cancer">0</td>
<td class="gt_row gt_right" headers="Active.Depression">0</td>
<td class="gt_row gt_right" headers="Hypercholesterolemia">1</td>
<td class="gt_row gt_right" headers="Hypertension">0</td>
<td class="gt_row gt_right" headers="Diabetes">0</td>
<td class="gt_row gt_right" headers="Liver.Disease">0</td>
<td class="gt_row gt_right" headers="Alcohol.Abuse">0</td>
<td class="gt_row gt_right" headers="Tobacco.Use.PastOrPresent">0</td>
<td class="gt_row gt_right" headers="Autoimmune.Disorder">0</td>
<td class="gt_row gt_right" headers="Thyroid.Disease">0</td>
<td class="gt_row gt_right" headers="Cardiovascular.Disease">0</td>
<td class="gt_row gt_right" headers="Reads">47370690</td>
<td class="gt_row gt_right" headers="Richness">85</td>
<td class="gt_row gt_right" headers="Shannon">2.692747</td>
<td class="gt_row gt_right" headers="Tax.PCoA1">-0.255345342</td>
<td class="gt_row gt_right" headers="Tax.PCoA2">0.09336676</td>
<td class="gt_row gt_right" headers="FBratio">3.5741204033301899</td>
<td class="gt_row gt_right" headers="Fnl.Richness">372</td>
<td class="gt_row gt_right" headers="Fnl.Shannon">5.246997</td>
<td class="gt_row gt_right" headers="Fnl.PCoA1">0.040150303</td>
<td class="gt_row gt_right" headers="Fnl.PCoA2">-0.009455487</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Run">SRR17648436</td>
<td class="gt_row gt_left" headers="Assay Type">WGS</td>
<td class="gt_row gt_right" headers="AvgSpotLen">243</td>
<td class="gt_row gt_right" headers="Bases">2727496965</td>
<td class="gt_row gt_left" headers="BioProject">PRJNA798058</td>
<td class="gt_row gt_left" headers="BioSample">SAMN25046610</td>
<td class="gt_row gt_left" headers="BioSampleModel">Metagenome or environmental</td>
<td class="gt_row gt_right" headers="Bytes">1004843427</td>
<td class="gt_row gt_left" headers="Center Name">WASHINGTON UNIVERSITY IN ST. LOUIS</td>
<td class="gt_row gt_right" headers="Collection_Date">2020</td>
<td class="gt_row gt_left" headers="Consent">public</td>
<td class="gt_row gt_left" headers="DATASTORE filetype">fastq,run.zq,sra</td>
<td class="gt_row gt_left" headers="DATASTORE provider">gs,ncbi,s3</td>
<td class="gt_row gt_left" headers="DATASTORE region">gs.us-east1,ncbi.public,s3.us-east-1</td>
<td class="gt_row gt_left" headers="Experiment">SRX13816621</td>
<td class="gt_row gt_left" headers="geo_loc_name_country">USA</td>
<td class="gt_row gt_left" headers="geo_loc_name_country_continent">North America</td>
<td class="gt_row gt_left" headers="geo_loc_name">USA: St. Louis</td>
<td class="gt_row gt_left" headers="HOST">Homo sapiens</td>
<td class="gt_row gt_left" headers="Instrument">Illumina NovaSeq 6000</td>
<td class="gt_row gt_left" headers="isolation_source">Stool 18</td>
<td class="gt_row gt_left" headers="lat_lon">38.6270 N 90.1994 W</td>
<td class="gt_row gt_left" headers="Library Name">Stool 18</td>
<td class="gt_row gt_left" headers="LibraryLayout">PAIRED</td>
<td class="gt_row gt_left" headers="LibrarySelection">RANDOM</td>
<td class="gt_row gt_left" headers="LibrarySource">METAGENOMIC</td>
<td class="gt_row gt_left" headers="Organism">human gut metagenome</td>
<td class="gt_row gt_left" headers="Platform">ILLUMINA</td>
<td class="gt_row gt_right" headers="ReleaseDate">2022-05-01</td>
<td class="gt_row gt_right" headers="create_date">2022-01-17 19:29:00</td>
<td class="gt_row gt_right" headers="version">1</td>
<td class="gt_row gt_right" headers="Sample Name">63218</td>
<td class="gt_row gt_left" headers="SRA Study">SRP355475</td>
<td class="gt_row gt_right" headers="Age">88.30411</td>
<td class="gt_row gt_right" headers="Sex">2</td>
<td class="gt_row gt_right" headers="Race">5</td>
<td class="gt_row gt_right" headers="Years Education">14</td>
<td class="gt_row gt_right" headers="CDR">0</td>
<td class="gt_row gt_left" headers="LUMIPULSE_CSF.ab42.ab40.ratio">9.1481640000000003E-2</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_tTau">204</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_pTau">29.3</td>
<td class="gt_row gt_left" headers="Days.SincePETpib">3338</td>
<td class="gt_row gt_left" headers="pib_fsuvr_rsf_tot_cortmean">1.01</td>
<td class="gt_row gt_left" headers="Centiloid.PiB">-2.0499999999999998</td>
<td class="gt_row gt_left" headers="Days.SincePETav45">NA</td>
<td class="gt_row gt_left" headers="av45_fsuvr_rsf_tot_cortmean">NA</td>
<td class="gt_row gt_left" headers="Centiloid.AV45">NA</td>
<td class="gt_row gt_left" headers="Tracer2Use">PiB</td>
<td class="gt_row gt_right" headers="combined.Centiloid">-2.0499999999999998</td>
<td class="gt_row gt_right" headers="amyloid.positive.AF">0</td>
<td class="gt_row gt_left" headers="Tauopathy">NA</td>
<td class="gt_row gt_right" headers="tau.positive.AF">0</td>
<td class="gt_row gt_right" headers="CortSig_Thickness">2.5991206619999998</td>
<td class="gt_row gt_right" headers="MR_LV_RV_HIPPOCAMPUS">6997.9</td>
<td class="gt_row gt_right" headers="WMH_volume">21365.375</td>
<td class="gt_row gt_right" headers="APOE">23</td>
<td class="gt_row gt_left" headers="APOE4.status">e4-</td>
<td class="gt_row gt_left" headers="Polygenic Risk Score">-1.5953700000000001E-2</td>
<td class="gt_row gt_right" headers="INT_PET">2567</td>
<td class="gt_row gt_right" headers="INT_CSF">1609</td>
<td class="gt_row gt_right" headers="INT_MRI">2562</td>
<td class="gt_row gt_right" headers="INT_CDR">111</td>
<td class="gt_row gt_left" headers="Interval_Measure_Amyloid">INT_PET</td>
<td class="gt_row gt_right" headers="Interval_Days_Amyloid">2567</td>
<td class="gt_row gt_right" headers="WEIGHT.lb">169.00000</td>
<td class="gt_row gt_right" headers="HEIGHT.in">61.00</td>
<td class="gt_row gt_right" headers="BMI">31.93194</td>
<td class="gt_row gt_right" headers="Cancer">0</td>
<td class="gt_row gt_right" headers="Active.Depression">0</td>
<td class="gt_row gt_right" headers="Hypercholesterolemia">0</td>
<td class="gt_row gt_right" headers="Hypertension">1</td>
<td class="gt_row gt_right" headers="Diabetes">0</td>
<td class="gt_row gt_right" headers="Liver.Disease">0</td>
<td class="gt_row gt_right" headers="Alcohol.Abuse">0</td>
<td class="gt_row gt_right" headers="Tobacco.Use.PastOrPresent">1</td>
<td class="gt_row gt_right" headers="Autoimmune.Disorder">0</td>
<td class="gt_row gt_right" headers="Thyroid.Disease">0</td>
<td class="gt_row gt_right" headers="Cardiovascular.Disease">0</td>
<td class="gt_row gt_right" headers="Reads">11190785</td>
<td class="gt_row gt_right" headers="Richness">81</td>
<td class="gt_row gt_right" headers="Shannon">3.319742</td>
<td class="gt_row gt_right" headers="Tax.PCoA1">0.102162752</td>
<td class="gt_row gt_right" headers="Tax.PCoA2">0.19155728</td>
<td class="gt_row gt_right" headers="FBratio">4.4321218886551002</td>
<td class="gt_row gt_right" headers="Fnl.Richness">265</td>
<td class="gt_row gt_right" headers="Fnl.Shannon">5.040629</td>
<td class="gt_row gt_right" headers="Fnl.PCoA1">-0.049224637</td>
<td class="gt_row gt_right" headers="Fnl.PCoA2">0.014985821</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Run">SRR17648356</td>
<td class="gt_row gt_left" headers="Assay Type">WGS</td>
<td class="gt_row gt_right" headers="AvgSpotLen">265</td>
<td class="gt_row gt_right" headers="Bases">3109979798</td>
<td class="gt_row gt_left" headers="BioProject">PRJNA798058</td>
<td class="gt_row gt_left" headers="BioSample">SAMN25046602</td>
<td class="gt_row gt_left" headers="BioSampleModel">Metagenome or environmental</td>
<td class="gt_row gt_right" headers="Bytes">1117200807</td>
<td class="gt_row gt_left" headers="Center Name">WASHINGTON UNIVERSITY IN ST. LOUIS</td>
<td class="gt_row gt_right" headers="Collection_Date">2020</td>
<td class="gt_row gt_left" headers="Consent">public</td>
<td class="gt_row gt_left" headers="DATASTORE filetype">fastq,run.zq,sra</td>
<td class="gt_row gt_left" headers="DATASTORE provider">gs,ncbi,s3</td>
<td class="gt_row gt_left" headers="DATASTORE region">gs.us-east1,ncbi.public,s3.us-east-1</td>
<td class="gt_row gt_left" headers="Experiment">SRX13816701</td>
<td class="gt_row gt_left" headers="geo_loc_name_country">USA</td>
<td class="gt_row gt_left" headers="geo_loc_name_country_continent">North America</td>
<td class="gt_row gt_left" headers="geo_loc_name">USA: St. Louis</td>
<td class="gt_row gt_left" headers="HOST">Homo sapiens</td>
<td class="gt_row gt_left" headers="Instrument">Illumina NovaSeq 6000</td>
<td class="gt_row gt_left" headers="isolation_source">Stool 10</td>
<td class="gt_row gt_left" headers="lat_lon">38.6270 N 90.1994 W</td>
<td class="gt_row gt_left" headers="Library Name">Stool 10</td>
<td class="gt_row gt_left" headers="LibraryLayout">PAIRED</td>
<td class="gt_row gt_left" headers="LibrarySelection">RANDOM</td>
<td class="gt_row gt_left" headers="LibrarySource">METAGENOMIC</td>
<td class="gt_row gt_left" headers="Organism">human gut metagenome</td>
<td class="gt_row gt_left" headers="Platform">ILLUMINA</td>
<td class="gt_row gt_right" headers="ReleaseDate">2022-05-01</td>
<td class="gt_row gt_right" headers="create_date">2022-01-17 19:22:00</td>
<td class="gt_row gt_right" headers="version">1</td>
<td class="gt_row gt_right" headers="Sample Name">62710</td>
<td class="gt_row gt_left" headers="SRA Study">SRP355475</td>
<td class="gt_row gt_right" headers="Age">79.84384</td>
<td class="gt_row gt_right" headers="Sex">2</td>
<td class="gt_row gt_right" headers="Race">5</td>
<td class="gt_row gt_right" headers="Years Education">16</td>
<td class="gt_row gt_right" headers="CDR">0</td>
<td class="gt_row gt_left" headers="LUMIPULSE_CSF.ab42.ab40.ratio">3.2471436999999999E-2</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_tTau">815</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_pTau">132.9</td>
<td class="gt_row gt_left" headers="Days.SincePETpib">1206</td>
<td class="gt_row gt_left" headers="pib_fsuvr_rsf_tot_cortmean">2.86</td>
<td class="gt_row gt_left" headers="Centiloid.PiB">81.2</td>
<td class="gt_row gt_left" headers="Days.SincePETav45">2632</td>
<td class="gt_row gt_left" headers="av45_fsuvr_rsf_tot_cortmean">1.909</td>
<td class="gt_row gt_left" headers="Centiloid.AV45">59.122399999999999</td>
<td class="gt_row gt_left" headers="Tracer2Use">PiB</td>
<td class="gt_row gt_right" headers="combined.Centiloid">81.2</td>
<td class="gt_row gt_right" headers="amyloid.positive.AF">1</td>
<td class="gt_row gt_left" headers="Tauopathy">1.3182499999999999</td>
<td class="gt_row gt_right" headers="tau.positive.AF">1</td>
<td class="gt_row gt_right" headers="CortSig_Thickness">2.4857915290000001</td>
<td class="gt_row gt_right" headers="MR_LV_RV_HIPPOCAMPUS">6178.1</td>
<td class="gt_row gt_right" headers="WMH_volume">10644.05566</td>
<td class="gt_row gt_right" headers="APOE">34</td>
<td class="gt_row gt_left" headers="APOE4.status">e4+</td>
<td class="gt_row gt_left" headers="Polygenic Risk Score">-2.9074700000000001E-3</td>
<td class="gt_row gt_right" headers="INT_PET">367</td>
<td class="gt_row gt_right" headers="INT_CSF">465</td>
<td class="gt_row gt_right" headers="INT_MRI">367</td>
<td class="gt_row gt_right" headers="INT_CDR">120</td>
<td class="gt_row gt_left" headers="Interval_Measure_Amyloid">INT_PET</td>
<td class="gt_row gt_right" headers="Interval_Days_Amyloid">367</td>
<td class="gt_row gt_right" headers="WEIGHT.lb">134.00000</td>
<td class="gt_row gt_right" headers="HEIGHT.in">64.30</td>
<td class="gt_row gt_right" headers="BMI">22.78669</td>
<td class="gt_row gt_right" headers="Cancer">0</td>
<td class="gt_row gt_right" headers="Active.Depression">0</td>
<td class="gt_row gt_right" headers="Hypercholesterolemia">1</td>
<td class="gt_row gt_right" headers="Hypertension">0</td>
<td class="gt_row gt_right" headers="Diabetes">0</td>
<td class="gt_row gt_right" headers="Liver.Disease">0</td>
<td class="gt_row gt_right" headers="Alcohol.Abuse">0</td>
<td class="gt_row gt_right" headers="Tobacco.Use.PastOrPresent">0</td>
<td class="gt_row gt_right" headers="Autoimmune.Disorder">0</td>
<td class="gt_row gt_right" headers="Thyroid.Disease">1</td>
<td class="gt_row gt_right" headers="Cardiovascular.Disease">0</td>
<td class="gt_row gt_right" headers="Reads">11712243</td>
<td class="gt_row gt_right" headers="Richness">93</td>
<td class="gt_row gt_right" headers="Shannon">3.510324</td>
<td class="gt_row gt_right" headers="Tax.PCoA1">0.198843509</td>
<td class="gt_row gt_right" headers="Tax.PCoA2">-0.01375146</td>
<td class="gt_row gt_right" headers="FBratio">4.5375031197984699</td>
<td class="gt_row gt_right" headers="Fnl.Richness">290</td>
<td class="gt_row gt_right" headers="Fnl.Shannon">5.078149</td>
<td class="gt_row gt_right" headers="Fnl.PCoA1">-0.024034290</td>
<td class="gt_row gt_right" headers="Fnl.PCoA2">0.043918188</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Run">SRR17648422</td>
<td class="gt_row gt_left" headers="Assay Type">WGS</td>
<td class="gt_row gt_right" headers="AvgSpotLen">269</td>
<td class="gt_row gt_right" headers="Bases">8818396225</td>
<td class="gt_row gt_left" headers="BioProject">PRJNA798058</td>
<td class="gt_row gt_left" headers="BioSample">SAMN25046596</td>
<td class="gt_row gt_left" headers="BioSampleModel">Metagenome or environmental</td>
<td class="gt_row gt_right" headers="Bytes">3467555540</td>
<td class="gt_row gt_left" headers="Center Name">WASHINGTON UNIVERSITY IN ST. LOUIS</td>
<td class="gt_row gt_right" headers="Collection_Date">2020</td>
<td class="gt_row gt_left" headers="Consent">public</td>
<td class="gt_row gt_left" headers="DATASTORE filetype">fastq,run.zq,sra</td>
<td class="gt_row gt_left" headers="DATASTORE provider">gs,ncbi,s3</td>
<td class="gt_row gt_left" headers="DATASTORE region">gs.us-east1,ncbi.public,s3.us-east-1</td>
<td class="gt_row gt_left" headers="Experiment">SRX13816635</td>
<td class="gt_row gt_left" headers="geo_loc_name_country">USA</td>
<td class="gt_row gt_left" headers="geo_loc_name_country_continent">North America</td>
<td class="gt_row gt_left" headers="geo_loc_name">USA: St. Louis</td>
<td class="gt_row gt_left" headers="HOST">Homo sapiens</td>
<td class="gt_row gt_left" headers="Instrument">Illumina NovaSeq 6000</td>
<td class="gt_row gt_left" headers="isolation_source">Stool 4</td>
<td class="gt_row gt_left" headers="lat_lon">38.6270 N 90.1994 W</td>
<td class="gt_row gt_left" headers="Library Name">Stool 4</td>
<td class="gt_row gt_left" headers="LibraryLayout">PAIRED</td>
<td class="gt_row gt_left" headers="LibrarySelection">RANDOM</td>
<td class="gt_row gt_left" headers="LibrarySource">METAGENOMIC</td>
<td class="gt_row gt_left" headers="Organism">human gut metagenome</td>
<td class="gt_row gt_left" headers="Platform">ILLUMINA</td>
<td class="gt_row gt_right" headers="ReleaseDate">2022-05-01</td>
<td class="gt_row gt_right" headers="create_date">2022-01-17 19:47:00</td>
<td class="gt_row gt_right" headers="version">1</td>
<td class="gt_row gt_right" headers="Sample Name">24882</td>
<td class="gt_row gt_left" headers="SRA Study">SRP355475</td>
<td class="gt_row gt_right" headers="Age">80.94795</td>
<td class="gt_row gt_right" headers="Sex">2</td>
<td class="gt_row gt_right" headers="Race">5</td>
<td class="gt_row gt_right" headers="Years Education">18</td>
<td class="gt_row gt_right" headers="CDR">0</td>
<td class="gt_row gt_left" headers="LUMIPULSE_CSF.ab42.ab40.ratio">6.7300982999999995E-2</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_tTau">396</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_pTau">49.3</td>
<td class="gt_row gt_left" headers="Days.SincePETpib">NA</td>
<td class="gt_row gt_left" headers="pib_fsuvr_rsf_tot_cortmean">NA</td>
<td class="gt_row gt_left" headers="Centiloid.PiB">NA</td>
<td class="gt_row gt_left" headers="Days.SincePETav45">1436</td>
<td class="gt_row gt_left" headers="av45_fsuvr_rsf_tot_cortmean">1.4510000000000001</td>
<td class="gt_row gt_left" headers="Centiloid.AV45">34.573599999999999</td>
<td class="gt_row gt_left" headers="Tracer2Use">AV45</td>
<td class="gt_row gt_right" headers="combined.Centiloid">34.573599999999999</td>
<td class="gt_row gt_right" headers="amyloid.positive.AF">1</td>
<td class="gt_row gt_left" headers="Tauopathy">1.47525</td>
<td class="gt_row gt_right" headers="tau.positive.AF">1</td>
<td class="gt_row gt_right" headers="CortSig_Thickness">2.482779727</td>
<td class="gt_row gt_right" headers="MR_LV_RV_HIPPOCAMPUS">6882.2</td>
<td class="gt_row gt_right" headers="WMH_volume">8009.0634769999997</td>
<td class="gt_row gt_right" headers="APOE">33</td>
<td class="gt_row gt_left" headers="APOE4.status">e4-</td>
<td class="gt_row gt_left" headers="Polygenic Risk Score">-4.3158299999999997E-3</td>
<td class="gt_row gt_right" headers="INT_PET">624</td>
<td class="gt_row gt_right" headers="INT_CSF">2268</td>
<td class="gt_row gt_right" headers="INT_MRI">624</td>
<td class="gt_row gt_right" headers="INT_CDR">93</td>
<td class="gt_row gt_left" headers="Interval_Measure_Amyloid">INT_PET</td>
<td class="gt_row gt_right" headers="Interval_Days_Amyloid">624</td>
<td class="gt_row gt_right" headers="WEIGHT.lb">97.39996</td>
<td class="gt_row gt_right" headers="HEIGHT.in">59.25</td>
<td class="gt_row gt_right" headers="BMI">19.50655</td>
<td class="gt_row gt_right" headers="Cancer">0</td>
<td class="gt_row gt_right" headers="Active.Depression">1</td>
<td class="gt_row gt_right" headers="Hypercholesterolemia">0</td>
<td class="gt_row gt_right" headers="Hypertension">1</td>
<td class="gt_row gt_right" headers="Diabetes">0</td>
<td class="gt_row gt_right" headers="Liver.Disease">0</td>
<td class="gt_row gt_right" headers="Alcohol.Abuse">0</td>
<td class="gt_row gt_right" headers="Tobacco.Use.PastOrPresent">0</td>
<td class="gt_row gt_right" headers="Autoimmune.Disorder">0</td>
<td class="gt_row gt_right" headers="Thyroid.Disease">0</td>
<td class="gt_row gt_right" headers="Cardiovascular.Disease">0</td>
<td class="gt_row gt_right" headers="Reads">32742673</td>
<td class="gt_row gt_right" headers="Richness">106</td>
<td class="gt_row gt_right" headers="Shannon">2.927976</td>
<td class="gt_row gt_right" headers="Tax.PCoA1">-0.095154022</td>
<td class="gt_row gt_right" headers="Tax.PCoA2">0.07741636</td>
<td class="gt_row gt_right" headers="FBratio">22.875371196652299</td>
<td class="gt_row gt_right" headers="Fnl.Richness">305</td>
<td class="gt_row gt_right" headers="Fnl.Shannon">4.967638</td>
<td class="gt_row gt_right" headers="Fnl.PCoA1">0.001006753</td>
<td class="gt_row gt_right" headers="Fnl.PCoA2">-0.017336432</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Run">SRR17648441</td>
<td class="gt_row gt_left" headers="Assay Type">WGS</td>
<td class="gt_row gt_right" headers="AvgSpotLen">246</td>
<td class="gt_row gt_right" headers="Bases">1988088952</td>
<td class="gt_row gt_left" headers="BioProject">PRJNA798058</td>
<td class="gt_row gt_left" headers="BioSample">SAMN25046605</td>
<td class="gt_row gt_left" headers="BioSampleModel">Metagenome or environmental</td>
<td class="gt_row gt_right" headers="Bytes">750726485</td>
<td class="gt_row gt_left" headers="Center Name">WASHINGTON UNIVERSITY IN ST. LOUIS</td>
<td class="gt_row gt_right" headers="Collection_Date">2020</td>
<td class="gt_row gt_left" headers="Consent">public</td>
<td class="gt_row gt_left" headers="DATASTORE filetype">fastq,run.zq,sra</td>
<td class="gt_row gt_left" headers="DATASTORE provider">gs,ncbi,s3</td>
<td class="gt_row gt_left" headers="DATASTORE region">gs.us-east1,ncbi.public,s3.us-east-1</td>
<td class="gt_row gt_left" headers="Experiment">SRX13816616</td>
<td class="gt_row gt_left" headers="geo_loc_name_country">USA</td>
<td class="gt_row gt_left" headers="geo_loc_name_country_continent">North America</td>
<td class="gt_row gt_left" headers="geo_loc_name">USA: St. Louis</td>
<td class="gt_row gt_left" headers="HOST">Homo sapiens</td>
<td class="gt_row gt_left" headers="Instrument">Illumina NovaSeq 6000</td>
<td class="gt_row gt_left" headers="isolation_source">Stool 13</td>
<td class="gt_row gt_left" headers="lat_lon">38.6270 N 90.1994 W</td>
<td class="gt_row gt_left" headers="Library Name">Stool 13</td>
<td class="gt_row gt_left" headers="LibraryLayout">PAIRED</td>
<td class="gt_row gt_left" headers="LibrarySelection">RANDOM</td>
<td class="gt_row gt_left" headers="LibrarySource">METAGENOMIC</td>
<td class="gt_row gt_left" headers="Organism">human gut metagenome</td>
<td class="gt_row gt_left" headers="Platform">ILLUMINA</td>
<td class="gt_row gt_right" headers="ReleaseDate">2022-05-01</td>
<td class="gt_row gt_right" headers="create_date">2022-01-17 19:21:00</td>
<td class="gt_row gt_right" headers="version">1</td>
<td class="gt_row gt_right" headers="Sample Name">63032</td>
<td class="gt_row gt_left" headers="SRA Study">SRP355475</td>
<td class="gt_row gt_right" headers="Age">81.70137</td>
<td class="gt_row gt_right" headers="Sex">2</td>
<td class="gt_row gt_right" headers="Race">5</td>
<td class="gt_row gt_right" headers="Years Education">16</td>
<td class="gt_row gt_right" headers="CDR">0</td>
<td class="gt_row gt_left" headers="LUMIPULSE_CSF.ab42.ab40.ratio">5.9846697999999997E-2</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_tTau">190</td>
<td class="gt_row gt_right" headers="LUMIPULSE_CSF_pTau">34.700000000000003</td>
<td class="gt_row gt_left" headers="Days.SincePETpib">961</td>
<td class="gt_row gt_left" headers="pib_fsuvr_rsf_tot_cortmean">2.7610000000000001</td>
<td class="gt_row gt_left" headers="Centiloid.PiB">76.745000000000005</td>
<td class="gt_row gt_left" headers="Days.SincePETav45">2059</td>
<td class="gt_row gt_left" headers="av45_fsuvr_rsf_tot_cortmean">2.4359999999999999</td>
<td class="gt_row gt_left" headers="Centiloid.AV45">87.369600000000005</td>
<td class="gt_row gt_left" headers="Tracer2Use">PiB</td>
<td class="gt_row gt_right" headers="combined.Centiloid">76.745000000000005</td>
<td class="gt_row gt_right" headers="amyloid.positive.AF">1</td>
<td class="gt_row gt_left" headers="Tauopathy">1.3015000000000001</td>
<td class="gt_row gt_right" headers="tau.positive.AF">1</td>
<td class="gt_row gt_right" headers="CortSig_Thickness">2.5486875329999998</td>
<td class="gt_row gt_right" headers="MR_LV_RV_HIPPOCAMPUS">6504.7000000000007</td>
<td class="gt_row gt_right" headers="WMH_volume">2848</td>
<td class="gt_row gt_right" headers="APOE">34</td>
<td class="gt_row gt_left" headers="APOE4.status">e4+</td>
<td class="gt_row gt_left" headers="Polygenic Risk Score">-3.7946299999999998E-3</td>
<td class="gt_row gt_right" headers="INT_PET">137</td>
<td class="gt_row gt_right" headers="INT_CSF">53</td>
<td class="gt_row gt_right" headers="INT_MRI">137</td>
<td class="gt_row gt_right" headers="INT_CDR">179</td>
<td class="gt_row gt_left" headers="Interval_Measure_Amyloid">INT_PET</td>
<td class="gt_row gt_right" headers="Interval_Days_Amyloid">137</td>
<td class="gt_row gt_right" headers="WEIGHT.lb">137.59998</td>
<td class="gt_row gt_right" headers="HEIGHT.in">66.00</td>
<td class="gt_row gt_right" headers="BMI">22.20899</td>
<td class="gt_row gt_right" headers="Cancer">0</td>
<td class="gt_row gt_right" headers="Active.Depression">0</td>
<td class="gt_row gt_right" headers="Hypercholesterolemia">0</td>
<td class="gt_row gt_right" headers="Hypertension">0</td>
<td class="gt_row gt_right" headers="Diabetes">0</td>
<td class="gt_row gt_right" headers="Liver.Disease">0</td>
<td class="gt_row gt_right" headers="Alcohol.Abuse">0</td>
<td class="gt_row gt_right" headers="Tobacco.Use.PastOrPresent">0</td>
<td class="gt_row gt_right" headers="Autoimmune.Disorder">0</td>
<td class="gt_row gt_right" headers="Thyroid.Disease">0</td>
<td class="gt_row gt_right" headers="Cardiovascular.Disease">0</td>
<td class="gt_row gt_right" headers="Reads">8067905</td>
<td class="gt_row gt_right" headers="Richness">80</td>
<td class="gt_row gt_right" headers="Shannon">3.400113</td>
<td class="gt_row gt_right" headers="Tax.PCoA1">0.078998610</td>
<td class="gt_row gt_right" headers="Tax.PCoA2">0.11305744</td>
<td class="gt_row gt_right" headers="FBratio">2.6135791440320402</td>
<td class="gt_row gt_right" headers="Fnl.Richness">253</td>
<td class="gt_row gt_right" headers="Fnl.Shannon">4.998603</td>
<td class="gt_row gt_right" headers="Fnl.PCoA1">-0.063743224</td>
<td class="gt_row gt_right" headers="Fnl.PCoA2">0.016006687</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
<section id="descarga-de-los-datos" class="level2">
<h2 class="anchored" data-anchor-id="descarga-de-los-datos">Descarga de los datos</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="ex">prefetch</span> SRR17648356</span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="ex">faster-dump</span> <span class="at">--split-files</span> SRR17648356/SRR17648356.sra</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a><span class="fu">gzip</span> SRR17648356_1.fastq</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a><span class="fu">gzip</span> SRR17648356_2.fastq</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="estructura-proyecto" class="level2">
<h2 class="anchored" data-anchor-id="estructura-proyecto">Estructura proyecto</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="ex">2025_Demo_Biobakery</span></span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── RUN_0_raw.sh</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── RUN_1_QC.sh</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── bin</span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── config.sh</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── log</span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── nohup.out</span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── old</span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> └── results</span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a>    <span class="ex">├──</span> 0_raw</span>
<span id="cb19-11"><a href="#cb19-11" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1.fastq.gz</span>
<span id="cb19-12"><a href="#cb19-12" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_2.fastq.gz</span>
<span id="cb19-13"><a href="#cb19-13" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648378_1.fastq.gz</span>
<span id="cb19-14"><a href="#cb19-14" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648378_2.fastq.gz</span>
<span id="cb19-15"><a href="#cb19-15" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648422_1.fastq.gz</span>
<span id="cb19-16"><a href="#cb19-16" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648422_2.fastq.gz</span>
<span id="cb19-17"><a href="#cb19-17" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648430_1.fastq.gz</span>
<span id="cb19-18"><a href="#cb19-18" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648430_2.fastq.gz</span>
<span id="cb19-19"><a href="#cb19-19" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648436_1.fastq.gz</span>
<span id="cb19-20"><a href="#cb19-20" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648436_2.fastq.gz</span>
<span id="cb19-21"><a href="#cb19-21" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648441_1.fastq.gz</span>
<span id="cb19-22"><a href="#cb19-22" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> └── SRR17648441_2.fastq.gz</span>
<span id="cb19-23"><a href="#cb19-23" aria-hidden="true" tabindex="-1"></a>    <span class="ex">├──</span> 1_QC</span>
<span id="cb19-24"><a href="#cb19-24" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.fastq.gz</span>
<span id="cb19-25"><a href="#cb19-25" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed.fastq.gz</span>
<span id="cb19-26"><a href="#cb19-26" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata.log</span>
<span id="cb19-27"><a href="#cb19-27" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata.repeats.removed.1.fastq</span>
<span id="cb19-28"><a href="#cb19-28" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata.repeats.removed.2.fastq</span>
<span id="cb19-29"><a href="#cb19-29" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_hg_39_bowtie2_paired_contam_1.fastq</span>
<span id="cb19-30"><a href="#cb19-30" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_hg_39_bowtie2_paired_contam_2.fastq</span>
<span id="cb19-31"><a href="#cb19-31" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_hg_39_bowtie2_unmatched_1_contam.fastq</span>
<span id="cb19-32"><a href="#cb19-32" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_hg_39_bowtie2_unmatched_2_contam.fastq</span>
<span id="cb19-33"><a href="#cb19-33" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_paired_1.fastq</span>
<span id="cb19-34"><a href="#cb19-34" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_paired_2.fastq</span>
<span id="cb19-35"><a href="#cb19-35" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_unmatched_1.fastq</span>
<span id="cb19-36"><a href="#cb19-36" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_cutadapt.trimmed_kneaddata_unmatched_2.fastq</span>
<span id="cb19-37"><a href="#cb19-37" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_1_discard.fastq</span>
<span id="cb19-38"><a href="#cb19-38" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_2_cutadapt.fastq.gz</span>
<span id="cb19-39"><a href="#cb19-39" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_2_cutadapt.trimmed.fastq.gz</span>
<span id="cb19-40"><a href="#cb19-40" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_2_discard.fastq</span>
<span id="cb19-41"><a href="#cb19-41" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> ├── SRR17648356_cutadapt.log</span>
<span id="cb19-42"><a href="#cb19-42" aria-hidden="true" tabindex="-1"></a>    <span class="ex">│&nbsp;&nbsp;</span> └── SRR17648356_trimmomatic.log</span>
<span id="cb19-43"><a href="#cb19-43" aria-hidden="true" tabindex="-1"></a>    <span class="ex">├──</span> 2_kraken2</span>
<span id="cb19-44"><a href="#cb19-44" aria-hidden="true" tabindex="-1"></a>    <span class="ex">├──</span> 3_bracken</span>
<span id="cb19-45"><a href="#cb19-45" aria-hidden="true" tabindex="-1"></a>    <span class="ex">└──</span> 4_humann</span>
<span id="cb19-46"><a href="#cb19-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-47"><a href="#cb19-47" aria-hidden="true" tabindex="-1"></a><span class="ex">├──</span> Miniconda3-latest-Linux-x86_64.sh</span>
<span id="cb19-48"><a href="#cb19-48" aria-hidden="true" tabindex="-1"></a><span class="ex">├──</span> bin</span>
<span id="cb19-49"><a href="#cb19-49" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── Bracken</span>
<span id="cb19-50"><a href="#cb19-50" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── Trimmomatic-0.39</span>
<span id="cb19-51"><a href="#cb19-51" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── automnt.sh</span>
<span id="cb19-52"><a href="#cb19-52" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── aws_install.sh</span>
<span id="cb19-53"><a href="#cb19-53" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── conda_install.sh</span>
<span id="cb19-54"><a href="#cb19-54" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── github_keyload.sh</span>
<span id="cb19-55"><a href="#cb19-55" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── github_setup.sh</span>
<span id="cb19-56"><a href="#cb19-56" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── keys</span>
<span id="cb19-57"><a href="#cb19-57" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── kraken2</span>
<span id="cb19-58"><a href="#cb19-58" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── pull-manifest.sh</span>
<span id="cb19-59"><a href="#cb19-59" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── set_repo.sh</span>
<span id="cb19-60"><a href="#cb19-60" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> └── sratoolkit.3.2.1-ubuntu64</span>
<span id="cb19-61"><a href="#cb19-61" aria-hidden="true" tabindex="-1"></a><span class="ex">├──</span> data</span>
<span id="cb19-62"><a href="#cb19-62" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── db_standard</span>
<span id="cb19-63"><a href="#cb19-63" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── humann_db</span>
<span id="cb19-64"><a href="#cb19-64" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── kneaddata_db</span>
<span id="cb19-65"><a href="#cb19-65" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> └── nohup.out</span>
<span id="cb19-66"><a href="#cb19-66" aria-hidden="true" tabindex="-1"></a><span class="ex">├──</span> log</span>
<span id="cb19-67"><a href="#cb19-67" aria-hidden="true" tabindex="-1"></a><span class="ex">├──</span> miniconda3</span>
<span id="cb19-68"><a href="#cb19-68" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── LICENSE.txt</span>
<span id="cb19-69"><a href="#cb19-69" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── _conda</span>
<span id="cb19-70"><a href="#cb19-70" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── bin</span>
<span id="cb19-71"><a href="#cb19-71" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── compiler_compat</span>
<span id="cb19-72"><a href="#cb19-72" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── conda-meta</span>
<span id="cb19-73"><a href="#cb19-73" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── condabin</span>
<span id="cb19-74"><a href="#cb19-74" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── envs</span>
<span id="cb19-75"><a href="#cb19-75" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── etc</span>
<span id="cb19-76"><a href="#cb19-76" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── include</span>
<span id="cb19-77"><a href="#cb19-77" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── lib</span>
<span id="cb19-78"><a href="#cb19-78" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── libexec</span>
<span id="cb19-79"><a href="#cb19-79" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── man</span>
<span id="cb19-80"><a href="#cb19-80" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── pkgs</span>
<span id="cb19-81"><a href="#cb19-81" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── share</span>
<span id="cb19-82"><a href="#cb19-82" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── shell</span>
<span id="cb19-83"><a href="#cb19-83" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── ssl</span>
<span id="cb19-84"><a href="#cb19-84" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── uninstall.sh</span>
<span id="cb19-85"><a href="#cb19-85" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> └── x86_64-conda-linux-gnu</span>
<span id="cb19-86"><a href="#cb19-86" aria-hidden="true" tabindex="-1"></a><span class="ex">├──</span> old</span>
<span id="cb19-87"><a href="#cb19-87" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── Bracken</span>
<span id="cb19-88"><a href="#cb19-88" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> ├── kraken2</span>
<span id="cb19-89"><a href="#cb19-89" aria-hidden="true" tabindex="-1"></a><span class="ex">│&nbsp;&nbsp;</span> └── miniconda_list.html</span>
<span id="cb19-90"><a href="#cb19-90" aria-hidden="true" tabindex="-1"></a><span class="ex">└──</span> snap</span>
<span id="cb19-91"><a href="#cb19-91" aria-hidden="true" tabindex="-1"></a>    <span class="ex">└──</span> tree</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="data" class="level2">
<h2 class="anchored" data-anchor-id="data">Data</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="ex">4.1G</span>    ./kneaddata_db</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a><span class="ex">93G</span> ./db_standard</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a><span class="ex">16G</span> ./humann_db/chocophlan</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a><span class="ex">34G</span> ./humann_db/uniref</span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a><span class="ex">2.6G</span>    ./humann_db/utility_mapping</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a><span class="ex">52G</span> ./humann_db</span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a><span class="ex">149G</span>    .</span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="runs" class="level2">
<h2 class="anchored" data-anchor-id="runs">RUNs</h2>
<ul>
<li>config.sh</li>
</ul>
<ul>
<li>RUN_0_raw.sh</li>
</ul>
<ul>
<li>RUN_1_QC.sh</li>
</ul>
</section>
<section id="preprocemiento" class="level2">
<h2 class="anchored" data-anchor-id="preprocemiento">Preprocemiento</h2>
<p>Con este paso quitamos ADN contaminante del hospedero y eliminamos secuencias adaptadoras.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="ex">kneaddata</span> <span class="at">-i1</span> SRR17648356_1.fastq.gz<span class="dt">\</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>-i2 SRR17648356_2.fastq.gz <span class="at">--output</span> out_kneaddata<span class="dt">\</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>--reference-db kneaddata_db <span class="at">--threads</span> 8<span class="dt">\</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>--trimmomatic-options=<span class="st">"LEADING:10"</span><span class="dt">\</span></span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>--trimmomatic-options=<span class="st">"TRAILING:10"</span><span class="dt">\</span></span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>--trimmomatic-options=<span class="st">"SLIDINGWINDOW:4:20"</span><span class="dt">\</span></span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>--trimmomatic-options=<span class="st">"MINLEN:60"</span> <span class="op">&gt;</span> <span class="va">$SRR17648356</span>-<span class="va">$(</span><span class="fu">date</span> +%s<span class="va">)</span>_kneaddata.log <span class="dv">2</span><span class="op">&gt;&amp;</span><span class="dv">1</span></span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Para evitar que la memoria se llene</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="co">#!/bin/bash</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Source the config file</span></span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a><span class="bu">source</span> config.sh</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Loop over each ID provided as argument</span></span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> file <span class="kw">in</span> <span class="st">"</span><span class="va">$@</span><span class="st">"</span><span class="kw">;</span> <span class="cf">do</span></span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Define basename based on the input pattern</span></span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>    <span class="va">basename</span><span class="op">=</span><span class="st">"</span><span class="va">${file}</span><span class="st">_1_cutadapt.trimmed"</span></span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Define the kept files (before gzipping)</span></span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a>    <span class="va">kept1</span><span class="op">=</span><span class="st">"</span><span class="va">${qcdir}</span><span class="st">/</span><span class="va">${basename}</span><span class="st">_kneaddata_paired_1.fastq"</span></span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a>    <span class="va">kept2</span><span class="op">=</span><span class="st">"</span><span class="va">${qcdir}</span><span class="st">/</span><span class="va">${basename}</span><span class="st">_kneaddata_paired_2.fastq"</span></span>
<span id="cb22-14"><a href="#cb22-14" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb22-15"><a href="#cb22-15" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Run kneaddata</span></span>
<span id="cb22-16"><a href="#cb22-16" aria-hidden="true" tabindex="-1"></a>    <span class="ex">kneaddata</span> <span class="dt">\</span></span>
<span id="cb22-17"><a href="#cb22-17" aria-hidden="true" tabindex="-1"></a>        <span class="at">-i1</span> <span class="st">"</span><span class="va">${qcdir}</span><span class="st">/</span><span class="va">${file}</span><span class="st">_1_cutadapt.trimmed.fastq.gz"</span> <span class="dt">\</span></span>
<span id="cb22-18"><a href="#cb22-18" aria-hidden="true" tabindex="-1"></a>        <span class="at">-i2</span> <span class="st">"</span><span class="va">${qcdir}</span><span class="st">/</span><span class="va">${file}</span><span class="st">_2_cutadapt.trimmed.fastq.gz"</span> <span class="dt">\</span></span>
<span id="cb22-19"><a href="#cb22-19" aria-hidden="true" tabindex="-1"></a>        <span class="at">--output</span> <span class="st">"</span><span class="va">${qcdir}</span><span class="st">"</span> <span class="dt">\</span></span>
<span id="cb22-20"><a href="#cb22-20" aria-hidden="true" tabindex="-1"></a>        <span class="at">--reference-db</span> <span class="st">"</span><span class="va">${dbs}</span><span class="st">/kneaddata_db"</span> <span class="dt">\</span></span>
<span id="cb22-21"><a href="#cb22-21" aria-hidden="true" tabindex="-1"></a>        <span class="at">--threads</span> <span class="st">"</span><span class="va">${threads}</span><span class="st">"</span> <span class="dt">\</span></span>
<span id="cb22-22"><a href="#cb22-22" aria-hidden="true" tabindex="-1"></a>        <span class="at">--bypass-trim</span> <span class="op">&gt;</span> <span class="st">"</span><span class="va">${logdir}</span><span class="st">/</span><span class="va">${file}</span><span class="st">-</span><span class="va">$(</span><span class="fu">date</span> +%s<span class="va">)</span><span class="st">_kneaddata.log"</span> <span class="dv">2</span><span class="op">&gt;&amp;</span><span class="dv">1</span></span>
<span id="cb22-23"><a href="#cb22-23" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb22-24"><a href="#cb22-24" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Compress the kept files</span></span>
<span id="cb22-25"><a href="#cb22-25" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gzip</span> <span class="st">"</span><span class="va">${kept1}</span><span class="st">"</span></span>
<span id="cb22-26"><a href="#cb22-26" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gzip</span> <span class="st">"</span><span class="va">${kept2}</span><span class="st">"</span></span>
<span id="cb22-27"><a href="#cb22-27" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb22-28"><a href="#cb22-28" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Define the compressed kept files</span></span>
<span id="cb22-29"><a href="#cb22-29" aria-hidden="true" tabindex="-1"></a>    <span class="va">kept1_gz</span><span class="op">=</span><span class="st">"</span><span class="va">${kept1}</span><span class="st">.gz"</span></span>
<span id="cb22-30"><a href="#cb22-30" aria-hidden="true" tabindex="-1"></a>    <span class="va">kept2_gz</span><span class="op">=</span><span class="st">"</span><span class="va">${kept2}</span><span class="st">.gz"</span></span>
<span id="cb22-31"><a href="#cb22-31" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb22-32"><a href="#cb22-32" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Delete all other files related to this ID</span></span>
<span id="cb22-33"><a href="#cb22-33" aria-hidden="true" tabindex="-1"></a>    <span class="fu">find</span> <span class="st">"</span><span class="va">${qcdir}</span><span class="st">"</span> <span class="at">-name</span> <span class="st">"</span><span class="va">${file}</span><span class="st">_*"</span> ! <span class="at">-name</span> <span class="st">"</span><span class="va">$(</span><span class="fu">basename</span> <span class="st">"</span><span class="va">${kept1_gz}</span><span class="st">"</span><span class="va">)</span><span class="st">"</span> ! <span class="at">-name</span> <span class="st">"</span><span class="va">$(</span><span class="fu">basename</span> <span class="st">"</span><span class="va">${kept2_gz}</span><span class="st">"</span><span class="va">)</span><span class="st">"</span> <span class="at">-delete</span></span>
<span id="cb22-34"><a href="#cb22-34" aria-hidden="true" tabindex="-1"></a><span class="cf">done</span></span></code><button title="Copiar al portapapeles" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copiado");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copiado");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




<script src="index_files/libs/quarto-html/zenscroll-min.js"></script>
</body></html>